common/saber_base.h

```diff
@@ -277,7 +287,8 @@ public:                                                         \
   enum ColorChangeMode {
     COLOR_CHANGE_MODE_NONE,
     COLOR_CHANGE_MODE_STEPPED,
-    COLOR_CHANGE_MODE_SMOOTH
+    COLOR_CHANGE_MODE_SMOOTH,
+    COLOR_CHANGE_MODE_ZOOMED,
```

Comment:
Seems like this needs an implementation somewhere?
Is it just in your prop file?

Reply:
I have the corresponding code in prop_base but you asked me to submit 1 file at a time

Reply:
it's supposed to be one *feature* at a time, not one file at a time.

Reply:
ah, sorry misunderstood.  ok let me test the define tomorrow and I'll get the corresponding code for prop_base added

Reply:
submitted prop_base addition